container/lxd/client.go

```diff
@@ -6,18 +6,28 @@ package lxd
 import (
 	"github.com/juju/errors"
 	"github.com/lxc/lxd/client"
+	"github.com/lxc/lxd/shared"
 )
 
-// Client extends the upstream LXD client.
+// Client extends the upstream LXD container server.
 type Client struct {
```

Comment:
Do we want to keep this called "Client"? It seems to be similar to the new LXD we should reference it as a Server. (It is the 'client' side of talking to a 'server'.)

Reply:
Had the same thoughts. I'll change it to Server.

Reply:
I might actually leave this for a new PR if you are happy to land this, as it is getting large already and we are looking to get on to the Bionic bugs.